DroneMaintenance.DAL/Configuration/UserConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DroneMaintenance.DAL.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace DroneMaintenance.DAL.Configuration
+{
+    class UserConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> builder)
+        {
+            builder.HasAlternateKey(u => u.Email);
+            builder.HasAlternateKey(u => u.Password);
+
+            builder.HasData
+            (
+                new User 
+                {
+                    Id = new Guid("85d2dad8-5ad5-4ad7-ab42-2a883cf0846a"),
+                    Name = "Sam",
+                    Email = "sam@email.com",
+                    Password = "admin",
```

Comment:
we shoud use hashed algoritms for password saving. For example you can use SimpleHash or KeyDerivation classes for this case.

Reply:
done